reportportal_client/service_async.py

```diff
@@ -284,6 +285,20 @@ def start_test_item(self, name, start_time, item_type, description=None,
         }
         self.queue.put_nowait(("start_test_item", args))
 
+    def update_test_item(self, description=None, tags=None):
+        """Updates test item
```

Comment:
This docstring does not correspond to PEP257.
D400, First line should end with a period. 
D401, First line should be in imperative mood.

Reply:
Fixed